src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -2982,28 +2982,45 @@ void FlexDRWorker::routeNet_postAstarPatchMinAreaVio(
     currArea = (minAreaConstraint) ? minAreaConstraint->getMinArea() : 0;
   }
   frCoord startViaHalfEncArea = 0, endViaHalfEncArea = 0;
-  FlexMazeIdx prevIdx = points[0], currIdx;
+  FlexMazeIdx currIdx = points[0], nextIdx;
   int i;
   int prev_i = 0;  // path start point
+  bool prev_is_wire = true;
```

Comment:
The terminology is getting mixed.  You changed prev/curr to curr/next but kept prev_i/prev_is_wire.

Reply:
I've kept prev_i and prev_is_wire because they're actually referring to the previous objects. If I understand the code correctly, the vars that I've changed to curr/next didn't have a good naming with prev/curr.